sky/serve/constants.py

```diff
@@ -70,3 +71,7 @@
 
 # Initial version of service.
 INITIAL_VERSION = 1
+
+# Placed here to solve circular import
+SKY_SERVE_CONTROLLER_NAME: str = (
+    f'sky-serve-controller-{common_utils.get_user_hash()}')
```

Comment:
iirc I once moved this constant from here to `serve_utils` to avoid circular import 😂 could you double check the circular import issue and resolve it?

Reply:
Well with the introduction of this line,
https://github.com/skypilot-org/skypilot/blob/acd996722b921db0eaeefbf365c8b845e419b209/sky/serve/serve_utils.py#L685
`serve_utils.py` needs to import from `controller_utils.py`. And if the `SKY_SERVE_CONTROLLER_NAME` constant is in `serve_utils.py`, `controller_utils.py` would need to import from `serve_utils.py` because of lines like 
https://github.com/skypilot-org/skypilot/blob/17adc606ff0ce7b6189ae383e1f5e829c1a87bf2/sky/utils/controller_utils.py#L92

How would you go about solving this circular import? The code does work with `SKY_SERVE_CONTROLLER_NAME` in `constants.py`